static/src/js/components/AdminRetrievals/__tests__/AdminRetrievalForm.test.jsx

```diff
@@ -1,29 +1,50 @@
-import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-import Form from 'react-bootstrap/Form'
+import { screen } from '@testing-library/react'
 
+import setupTest from '../../../../../../jestConfigs/setupTest'
 import { AdminRetrievalsForm } from '../AdminRetrievalsForm'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup() {
-  const props = {
-    onAdminViewRetrieval: jest.fn()
+const setup = setupTest({
+  Component: AdminRetrievalsForm,
+  defaultProps: {
+    onAdminViewRetrieval: jest.fn(),
+    onFetchAdminRetrievals: jest.fn()
   }
+})
+
+describe('AdminRetrievalsForm component', () => {
+  test('renders itself correctly', () => {
+    setup()
+  })
 
-  const enzymeWrapper = shallow(<AdminRetrievalsForm {...props} />)
+  test('onInputChange updates state', async () => {
```

Comment:
If you look at the tests in EDSC you won't always see good examples of this, but we write tests like this now:

describe: describe what the user interaction is on the page, or the input to the test
    test: describe the expected outcome, or result

So something like this:
```
describe('when typing into the retrieval id field', () => {
    test('shows the typed text in the input', () => {
```

Using this format greatly helps future devs understand what exactly the test is supposed to be doing, and what the expected result of the test should be. And in turn that helps us understand what the intended functionality of the code is. Each test is this file should be updated using this format